LogMonitor/src/LogMonitor/EtwMonitor.cpp

```diff
@@ -748,7 +754,7 @@ EtwMonitor::PrintEvent(
         }
 
         std::wstring formattedEvent = Utility::FormatString(
-            L"<Source>EtwEvent</Source>%ls%ls",
+            L"<EventLog><Source>EtwEvent</Source>%ls%ls</EventLog>",

```

Comment:
The wording could be confusing, since also have `EventLog` as a source type in - [`EventMonitor`](https://github.com/microsoft/windows-container-tools/blob/main/LogMonitor/src/LogMonitor/EventMonitor.cpp)

Reply:
This is now changed to Event

Reply:
I've also added the Preamble Tag as settings in the latest push

Logs with the setting turn on: `"includePreambleTag": true,`

`<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Event
	xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
	<Source>EtwEvent</Source>
	<Time>2022-12-21T17:40:00.000Z</Time>
	<Provider idGuid="{DAA6A96B-F3E7-4D4D-A0D6-31A350E6A445}"/>
	<DecodingSource>DecodingSourceXMLFile</DecodingSource>
	<Execution ProcessID="9076" ThreadID="26912" />
	<Level>Information</Level>
	<Keyword>0x8000000000000001</Keyword>
	<EventID Qualifiers="0">0</EventID>
	<EventData>
		<FrameUniqueID>5447627</FrameUniqueID>
		<PortNumber>0</PortNumber>
		<TID>0</TID>
		<PeerID>0</PeerID>
		<PayloadLength>95</PayloadLength>
		<QueueLength>0</QueueLength>
		<QueueState>false</QueueState>
		<CustomData1>24</CustomData1>
		<CustomData2>0</CustomData2>
		<CustomData3>0</CustomData3>
	</EventData>
</Event>
<Event
	xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
	<Source>EtwEvent</Source>
	<Time>2022-12-21T17:40:00.000Z</Time>
	<Provider idGuid="{DAA6A96B-F3E7-4D4D-A0D6-31A350E6A445}"/>
	<DecodingSource>DecodingSourceXMLFile</DecodingSource>
	<Execution ProcessID="9076" ThreadID="26912" />
	<Level>Information</Level>
	<Keyword>0x8000000000000001</Keyword>
	<EventID Qualifiers="1">1</EventID>
	<EventData>
		<FrameUniqueID>5447627</FrameUniqueID>
		<QueueLength>0</QueueLength>
		<QueueState>false</QueueState>
		<Status>0</Status>
		<CustomData1>0</CustomData1>
		<CustomData2>0</CustomData2>
		<CustomData3>0</CustomData3>
	</EventData>
</Event>`


Logs with the setting turn off: `"includePreambleTag": false,`

`<Event
	xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
	<Source>EtwEvent</Source>
	<Time>2022-12-21T17:43:30.000Z</Time>
	<Provider idGuid="{DAA6A96B-F3E7-4D4D-A0D6-31A350E6A445}"/>
	<DecodingSource>DecodingSourceXMLFile</DecodingSource>
	<Execution ProcessID="30472" ThreadID="17488" />
	<Level>Information</Level>
	<Keyword>0x8000000000000001</Keyword>
	<EventID Qualifiers="0">0</EventID>
	<EventData>
		<FrameUniqueID>5465690</FrameUniqueID>
		<PortNumber>0</PortNumber>
		<TID>0</TID>
		<PeerID>0</PeerID>
		<PayloadLength>49</PayloadLength>
		<QueueLength>0</QueueLength>
		<QueueState>false</QueueState>
		<CustomData1>24</CustomData1>
		<CustomData2>0</CustomData2>
		<CustomData3>0</CustomData3>
	</EventData>
</Event>
<Event
	xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
	<Source>EtwEvent</Source>
	<Time>2022-12-21T17:43:30.000Z</Time>
	<Provider idGuid="{DAA6A96B-F3E7-4D4D-A0D6-31A350E6A445}"/>
	<DecodingSource>DecodingSourceXMLFile</DecodingSource>
	<Execution ProcessID="30472" ThreadID="17488" />
	<Level>Information</Level>
	<Keyword>0x8000000000000001</Keyword>
	<EventID Qualifiers="1">1</EventID>
	<EventData>
		<FrameUniqueID>5465690</FrameUniqueID>
		<QueueLength>0</QueueLength>
		<QueueState>false</QueueState>
		<Status>0</Status>
		<CustomData1>0</CustomData1>
		<CustomData2>0</CustomData2>
		<CustomData3>0</CustomData3>
	</EventData>
</Event>`

Reply:
Also, some work is needed on the tests!